library/ui-styles/src/main/res/values/theme_dark.xml

```diff
@@ -134,6 +134,8 @@
         <item name="vctr_social_login_button_gitlab_style">@style/Widget.Vector.Button.Outlined.SocialLogin.Gitlab.Dark</item>
 
         <item name="vctr_jump_to_unread_style">@style/Widget.Vector.JumpToUnread.Dark</item>
+        <!-- Keywords -->
+        <item name="vctr_keyword_style">@style/Widget.Vector.Keyword</item>
```

Comment:
Since the value is the same for light and dark theme, there is no need to declare `vctr_keyword_style`. You can directly use `Widget.Vector.Keyword`